tools/nni_cmd/launcher_utils.py

```diff
@@ -269,6 +273,9 @@ def validate_pai_trial_conifg(experiment_config):
             print_warning(warning_information.format('dataDir'))
         if experiment_config.get('trial').get('outputDir'):
             print_warning(warning_information.format('outputDir'))
+    if experiment_config.get('trainingServicePlatform') == 'pai':
+        if experiment_config.get('trial').get('paiConfigPath'):
+            get_yml_content(experiment_config['trial']['paiConfigPath'])
```

Comment:
what is the meaning of this line? unused return content

Reply:
validate the format of yaml file